recipes/mcxtrace-suite/meta.yaml

```diff
@@ -0,0 +1,134 @@
+{% set version = "3.4.9" %}
+{% set sha_value = "ccab20cca6e842d5b643ae30c0cd3330c3af31375b205f251cfafc086fd8000b" %}
+#NB get sha256 with: curl -sL <URL> | openssl sha256
```

Comment:
```suggestion
```

Reply:
Thanks @ocefpaf, resolved in https://github.com/conda-forge/staged-recipes/pull/25503/commits/4367e8bedb8752fc87c1beddbe951c708dfd5540